### PR TITLE
makefile: fix rbdmap manpage

### PR DESCRIPTION
**Pull request by tchaikov**
fix the build failure at 
- http://gitbuilder.sepia.ceph.com/gitbuilder-ceph-deb-trusty-amd64-notcmalloc/log.cgi?log=a431d04acb0c5249561761fdbf2aedd068d4a858
- http://gitbuilder.sepia.ceph.com/gitbuilder-ceph-rpm-centos7-amd64-basic/log.cgi?log=a431d04acb0c5249561761fdbf2aedd068d4a858

### CODE REVIEW

**Comment by tchaikov** [comment]
pushed to gitbuilder also for testing.

File: man/CMakeLists.txt [hunk 1]
```diff
@@ -17,17 +16,28 @@ install(FILES
...
+if(WITH_RBD)
+  install(FILES
+	ceph-rbdnamer.8
 	rbd-nbd.8
-	rbd-replay.8
 	rbd-replay-prep.8
+	rbd-replay.8
 	rbdmap.8
-  DESTINATION ${CEPH_MAN_DIR}/man8)
+    rbd.8
```
Comment (smithfarm) [inline man/CMakeLists.txt]: increase indent here, so it lines up with `rbdmap.8` ?

Comment (tchaikov) [inline man/CMakeLists.txt]: oh, tab versus spaces.  tab stop = 4, fixing.

**Comment by tchaikov** [comment]
changelog

- use tab instead of spaces for indent.

**Comment by liewegas** [comment]
lgtm, merge once gitbuilders are happy?

**Comment by tchaikov** [comment]
gitbuilders are almost happy, we still need #8293 to appease them.

**Comment by tchaikov** [comment]
```
Checking for unpackaged file(s): /usr/lib/rpm/check-files /srv/autobuild-ceph/gitbuilder.git/build/rpmbuild/BUILDROOT/ceph-10.0.5-2814.gc1efc63.el7.x86_64
error: Installed (but unpackaged) file(s) found:
/etc/cron.hourly/subman.pyc
/etc/cron.hourly/subman.pyo
RPM build errors:
Installed (but unpackaged) file(s) found:
/etc/cron.hourly/subman.pyc
/etc/cron.hourly/subman.pyo
```

**Pull request merged**